test/unit/account/search_test.rb

```diff
@@ -65,6 +65,22 @@ class Account::SearchTest < ActiveSupport::TestCase
     assert_equal 'fo\/o', Account.search_ids('fo/o').query
   end
 
+  test 'search with query by email' do
+    ThinkingSphinx::Test.run do
+     provider = FactoryBot.create(:provider_account)
```

Comment:
```suggestion
     provider = FactoryBot.create(:simple_provider)
```
For efficiency 😄 Because `simple_provider` creates a provider but `provider_account` creates much more that you do not need for this test ;) like contracts and stuff that you are not using here.